### ISSUE TITLE
Text error in Dutch version (WCAG-EM)

### ISSUE THREAD

**Issue by julezrulez** [w3c/wai-wcag-em-report-tool#202]
**Describe the bug**
In the Dutch text it says "Het is gebaseerd op de Web Content Accessibility Guidelines Evaluation Methodology (WCAG-EM)."

**To Reproduce**
1. Go to https://www.w3.org/WAI/eval/report-tool/
2. Click on 'Nederlands'
3. Look into the intro text

**Expected behavior**
"Web Content Accessibility Guidelines Evaluation Methodology " should be "Website Accessibility Conformance Evaluation Methodology".
This might also going wrong at other places in the text. 

Same problem might appear in other language. It is also wrong in English.
